tasks/mc/data.py

```diff
@@ -108,7 +108,7 @@ def geoname_format(country, name):
                        metro_level_name=METRO_LEVEL_NAME.get(country, 'metro'))
 
 
-HOST = 'http://172.18.0.1:8000/mc'
+HOST = 'http://172.17.0.1:8000/mc'
```

Comment:
Intended change?

Reply:
Yes. 99% of the times Docker assigns `172.17.0.0` subnetwork.